firmware/src/rdm.h

```diff
@@ -265,6 +265,14 @@ typedef enum {
 
 
 /**
+ * @brief Open Lighting Project manufacturer PIDs.
+ */
+typedef enum {
+  PID_DEVICE_MODEL = 0x8002,
```

Comment:
I know we've failed so far, but I wonder if we should try and follow this from RDM 6.2.10.2 for these and future PIDs "Manufacturer-Specific PID values should be selected by choosing the appropriate category from Table A-3 and adding an offset of 0x8000 to preserve a logical organization."